frontend/src/routes/project/[project_code]/+page.svelte

```diff
@@ -34,21 +38,34 @@
 		});
 	}
 
-	const canManage = $user?.role == 'admin' || $user?.projects.find(project => project.code == project.code)?.role == 'Manager';
+	async function updateProjectName(newName: string) {
+		return _changeProjectName({ projectId: _project.id, name: newName });
+	}
+
+	async function updateProjectDescription(newDescription: string) {
+		return _changeProjectDescription({ projectId: _project.id, description: newDescription });
+	}
+
+	$: userId = $user?.id;
+	$: isAdmin = $user?.role == 'admin';
+	$: canManage =
+		isAdmin || $user?.projects.find((project) => project.code == project.code)?.role == 'Manager';
 
 	const projectNameValidation = z.string().min(1, $t('project_page.project_name_empty_error'));
 </script>
 
 <div class="space-y-4">
 	{#if project}
-		<div class="text-3xl flex items-center gap-3 flex-wrap">
-			<span>{$t('project_page.project')}:</span>
-			<EditableText
-				disabled={!canManage}
-				value={project.name}
-				validation={projectNameValidation}
-				saveHandler={(newValue) => _changeProjectName({ projectId: _project.id, name: newValue })}
-			/>
+		<div class="space-y-2">
+			<div class="text-3xl flex items-center gap-3 flex-wrap">
+				<span>{$t('project_page.project')}:</span>
+				<EditableText
+					disabled={!canManage}
+					bind:value={project.name}
+					validation={projectNameValidation}
```

Comment:
idea for later might be to add a `required` attribute that would do the same, optionally accepting the error text.